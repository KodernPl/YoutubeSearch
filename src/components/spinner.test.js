import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Spinner from './spinner';

const renderComponent = () => {
  return shallow((
    <Spinner />
  ));
};

test('renders correctly', () => {
  const component = renderComponent();

  expect(toJson(component)).toMatchSnapshot();
});
