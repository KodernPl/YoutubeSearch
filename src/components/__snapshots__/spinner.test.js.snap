// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<spinner__SpinnerWrapper>
  Loading...
</spinner__SpinnerWrapper>
`;
